v2/qr_code.go

```diff
@@ -0,0 +1,20 @@
+package omise
+
+// QrCode represents resource object.
+type QrCode struct {
+	Base
+	Email string `json:"email"`
+	MfaProvisioningURI string `json:"mfa_provisioning_uri"`
```

Comment:
`Mfa` here should be `MFA` or there's some limitations that prevent this kind of naming?